components/Banners.tsx

```diff
@@ -0,0 +1,56 @@
+import Link from "next/link";
+import React from "react";
+
+const items: string[][] = [
+  ["Ser Parceiro", "Parceiros", "Equipa"],
+  ["Sobre Nós"],
+  ["Departamento de Informática", "Estudar LEI"],
+];
+
+interface RowProps {
+  links: string[];
+  rowIndex: number;
+}
+
+function handleColSpan(rowIndex: number, colIndex: number): number {
+  if (rowIndex === 0) {
+    return colIndex === 0 ? 2 : 1;
+  } else if (rowIndex === 1) {
+    return 4;
+  } else if (rowIndex === 2) {
+    return 2;
+  } else {
+    return 1;
+  }
+}
```

Comment:
tailwind advises against constructing class names dynamically, instead, return the complete class names.

see [https://tailwindcss.com/docs/content-configuration#dynamic-class-names](url)